<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(166, 199, 171);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Retro Computer&quot;;
background-color: rgb(209, 209, 209);</string>
    </property>
    <property name="text">
     <string>QUIT GAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(209, 209, 209);
font: 700 12pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>530</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);
background-color: rgb(209, 209, 209);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(209, 209, 209);
color: rgb(161, 161, 0);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 0);
background-color: rgb(209, 209, 209);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputConsole">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>10</y>
      <width>381</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#outputConsole{
background-color: transparent;
border-image: url(:title.png);
background: none;
border: none;
background-repeat: none;
}
#upLeft:pressed
{
	border-image: url(:title.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="gameText">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>511</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(196, 220, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="telButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
font: 10pt &quot;Retro Computer&quot;;
background-color: rgb(209, 209, 209);</string>
    </property>
    <property name="text">
     <string>Teleport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Retro Computer&quot;;
background-color: rgb(209, 209, 209);</string>
    </property>
    <property name="text">
     <string>Help!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="mapBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 223, 235);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mapButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mapButton{
background-color: transparent;
border-image: url(:map2.png);
background: none;
border: none;
background-repeat: none;
}
#upLeft:pressed
{
	border-image: url(:map2.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>-10</y>
      <width>511</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
